stdlib/public/stubs/CommandLine.cpp

```diff
@@ -242,6 +242,39 @@ char ** _swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
 
   return argv;
 }
+#elif defined(__OpenBSD__)
+#include <sys/types.h>
+#include <sys/sysctl.h>
+#include <sys/exec.h>
+
+SWIFT_RUNTIME_STDLIB_API
+char ** _swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
+  if (_swift_stdlib_ProcessOverrideUnsafeArgv) {
+    *outArgLen = _swift_stdlib_ProcessOverrideUnsafeArgc;
+    return _swift_stdlib_ProcessOverrideUnsafeArgv;
+  }
+
+  int mib[2] = {CTL_VM, VM_PSSTRINGS};
+  struct _ps_strings _ps;
```

Comment:
I'm unfamiliar with the platform requirements of OpenBSD, but do we need to conditionalize this on `VM_PSSTRINGS` given your current platform constraints?

Reply:
If I understand you correctly, there is no need as this should always be available (this is like e.g., the FreeBSD implementation with `KERN_PROC_ARGS`, also see https://man.openbsd.org/sysctl.2#VM_PSSTRINGS~2 )